tests/libnet/cluster.go

```diff
@@ -0,0 +1,38 @@
+package libnet
+
+import (
+	"fmt"
+
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	netutils "k8s.io/utils/net"
+
+	"kubevirt.io/client-go/kubecli"
+)
+
+func IsClusterDualStack(virtClient kubecli.KubevirtClient) (bool, error) {
+	// grab us some neat kubevirt pod; let's say virt-handler is our target.
+	targetPodType := "virt-handler"
+	virtHandlerPod, err := getPodByKubeVirtRole(virtClient, targetPodType)
+	if err != nil {
+		return false, err
+	}
+	for _, ip := range virtHandlerPod.Status.PodIPs {
```

Comment:
My concern here is ipv6 single stack cluster.

Reply:
We do not support (document and test) single-stack IPv6. If somebody uses it, it may work, but it is an undefined behavior we are not focused at right now.

Reply:
Although we don't support single stack ipv6, I think that for the completeness and readability of the test it worth checking the `PodIPs` list has 2 ips. But won't insist on it.

Reply:
we do support single stack (which is not ipv6), on 1.17 so if we enforce 2 ips, the 1.17 lane would fail
or am i miss something?
we never enforce 2 ips, we just sometimes check there is at least 1 ip,
but here its the pod list, not the vmi, and the pod list is mandatory to have at least 1 ip by k8s itself

Reply:
I think its a must, at least as long as we have kind,
and even without kind, i think we indeed should because this function is "IsClusterDualStack"
any just 1 ipv6 is not dual stack sufficient condition

Reply:
Also what if the cluster is broken, it wont detect it, and this is the point of the test isnt it ?

Reply:
I don't want to add a single line of code whose goal is to be consistent with kind, since our objective is getting rid of it.

Never the less, I'll need to do it to keep the method consistent with its name.

Reply:
Done (checked if there are more than 1 addresses on the main pod iface).

Reply:
you can check exactly 2 imo in order to be more accurate, 
because there are at most 1 per family (k8s enforcement) and there is just 2 families atm on dual stack.